src/Google/Service/Resource.php

```diff
@@ -232,6 +200,37 @@ public function call($name, $arguments, $expectedClass = null)
     return $this->client->execute($request, $expectedClass);
   }
 
+  private function logServiceError($txt, $name, $parameter = null)
```

Comment:
Let's make the method signature look like this:

```
private function log($level, $txt, $name, $parameter = null)
```

Then we can make "info" and "error" the first argument to these calls, and consolidate two methods into one.